bazel/dependencies/libz.BUILD.bazel

```diff
@@ -0,0 +1,41 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+filegroup(
+    name = "libz_all",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "libz_headers",
+    srcs = glob(["**/*.h"]),
+    visibility = ["//visibility:public"],
+)
+
+
+# zlib
+# The name of this rule is libz to avoid conflicts with the system `@zlib//:zlib`.
+# We cannot use @zlib//:zlib because the libraries that are generated are named
+# libzlib.(a|so), which is incompatible with other foreign_cc builds.
+cmake(
+    name = "libz", 
+    env = select({
+        "@//bazel/platforms:e1_gnu_build" : {
+            "CHOST": "arm",
+            "LDFLAGS": "-fuse-ld=gold",
```

Comment:
Hopefully we can remove this once my `rules_foreign_cc` patch goes in?
https://github.com/enfabrica/enkit/pull/1129

Also curious about `CHOST` arg.

Reply:
https://github.com/madler/zlib/blob/d476828316d05d54c6fd6a068b121b30c147b5cd/configure#L34

`CHOST` might not be needed since I changed over to cmake build. I don't see it used there.

Reply:
I will test with your change.